src/Modules/Events/EventCardComponent.js

```diff
@@ -47,16 +49,26 @@ const EventCardComponent = (props) => {
             <br />
             {eventCity} {eventState} {eventZip}<br />
             {phoneNumber}
+            <br /><br />
           </div>
+          {showDetails && <div className="">
+            <p>
+              <b> Information </b>
+              <br />
+              {eventDetails}
+            </p>
+          </div>}
           {/* Out of scope */}
```

Comment:
Can we drop the comment line now? With this PR it will now be in-scope right?

Reply:
removed.